disnake/message.py

```diff
@@ -896,6 +896,10 @@ def __init__(
         except AttributeError:
             self.guild = state._get_guild(utils._get_as_snowflake(data, "guild_id"))
 
+        if thread_data := data.get("thread"):
```

Comment:
This should be added to `.types.message._MessageOptional` as well

Reply:
https://github.com/DisnakeDev/disnake/pull/291/commits/b5da05bf5c9ce9460c8a28096400a3ebe1cb4402